TLM/TLM/UI/SubTools/JunctionRestrictionsTool.cs

```diff
@@ -378,7 +382,7 @@ public class JunctionRestrictionsTool : SubTool {
 			var zoom = 1.0f / diff.magnitude * 100f * MainTool.GetBaseZoom();
 			var size = (small ? 0.75f : 1f) * (viewOnly ? 0.8f : 1f) * junctionRestrictionsSignSize * zoom;
 
-			var boundingBox = new Rect(signScreenPos.x - size / 2, signScreenPos.y - size / 2, size, size * aspectRatio);
```

Comment:
the parameter `aspectRatio` is always 1.0

Reply:
I think I didn't noticed that while reviewing

Reply:
This feature was added due to the sign being a different aspect ratio before hand. Would it not be nice to keep it in case we end up having a non-proportional sign in the future? It's better to keep our options open, no?